AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/CredentialStorage/AmplifyCredentials.swift

```diff
@@ -7,10 +7,10 @@
 
 import Foundation
 
-struct CognitoCredentials: Codable {
+struct AmplifyCredentials: Codable {
```

Comment:
This is just for internal representation right?

Reply:
Yes.